src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -761,7 +761,7 @@ namespace UtilityRoutines {
                     " Severe Errors; Elapsed Time=" + Elapsed);
         DisplayString("EnergyPlus Run Time=" + Elapsed);
 
-        {
+        if (state.files.outputControl.end) {
             auto tempfl = state.files.endFile.try_open(state.files.outputControl.end);
```

Comment:
I think the true issue is with `try_open()`. It has a similar issue to the #8236. It should be `file.open(false, output_to_file);` in IOFiles.hh. You should not need the enclosing if statement at that point. The current code will still have an issue with the END file even with this change.

Reply:
Ok, I went ahead and fixed this. I haven't tested it yet since I have to run out but it should work.